fe/fe-core/src/main/java/org/apache/doris/nereids/rules/exploration/mv/MaterializedViewUtils.java

```diff
@@ -472,6 +474,13 @@ public Void visitLogicalRelation(LogicalRelation relation, IncrementCheckerConte
                 return null;
             }
             Column mvReferenceColumn = contextPartitionColumn.getColumn().get();
+            Expr definExpr = mvReferenceColumn.getDefineExpr();
```

Comment:
could u add some comment to explain this code? rbo rewrite for sync mv will be removed in future, comment could help to remove this related code too